accepted/2024/net-standard-recommendation.md

```diff
@@ -0,0 +1,128 @@
+# .NET Standard Targeting Recommendations
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst) | [Viktor Hofer](https://github.com/ViktorHofer)
+
+We have already said that there is [no new version of .NET
+Standard][net-standard-future] and that the replacement moving forward is to
+just target the platform-neutral `netX.Y` [base framework][tfm] (with `x.y >=
+5.0`) of .NET Core.
+
+For customers that need to author code that still needs to be consumed by .NET
+Framework, the recommendation is to continue using `netstandard2.0`. There is
+very little reason to target .NET Standard 2.1 because you lose .NET Framework
+while only gaining very little few additional APIs. And if you don't care about
+.NET Framework support, then just target .NET Core directly; at least then you
+get access to a lot more functionality, in exchange for the reduced reach.
+
+There is basically no reason to target .NET Standard 1.x any more as all .NET
```

Comment:
I think it would be worth mentioning some of the drawbacks of targeting .NET Standard 1.x.  You get a LOT of packages downloaded/in your graph, 73 by my count without a RuntimeIdentifier specified, and 103 if you do specify a RuntimeIdentifier.

Reply:
I believe I have addressed this in 83157fdf99fb4e472cb9ef024273837ee73054e1.